DiscordBot/Commands/EventCommands.cs

```diff
@@ -14,6 +14,7 @@ internal class EventCommands
         private static readonly string[] EventOperations =
         {
             "create",
+            "remove",
             "rm",
```

Comment:
What's `rm` for? 🙂

Reply:
It was the previous keyword before renaming the command to remove, but I thought I'd leave it in there as an option.

Should I remove this?

Reply:
I would definitely recommend removing it if it's not used! 🙂